maven-core/src/main/java/org/apache/maven/lifecycle/internal/ReactorContext.java

```diff
@@ -73,11 +68,4 @@ public ClassLoader getOriginalContextClassLoader()
         return originalContextClassLoader;
     }
 
-    /**
-     * @since 3.3.0
-     */
-    public SessionScope.Memento getSessionScopeMemento()
```

Comment:
This should rather be deprecated in 3.9.0 and removed in 4.0, no?

Reply:
ReactorContext is not a public API, so I don't see why we should go through a deprecation stage here.

Reply:
True